search-sentiments-map/public/style.css

```diff
@@ -15,13 +21,23 @@ body {
   margin: 20px;
 }
 
+#trends-list li {
+  font-family: "Lucida Console", Courier, monospace;
+}
+
+#trends-list li:hover {
+  cursor: pointer;
+  color: #2268d6;
+}
+

```

Comment:
This is a general comment, but it seems that when you click on a trend all this does is update the heading of the page. Rather than manually changing CSS and having an onClick function, consider swapping to links instead(will handle CSS) and using routing to update the heading,  so when we add in the click to view trend map, it can be done within that routing as well.

Reply:
Code should be added to display the sentiments when you click on a trend. But since we don't have sentiment scores I'm not doing it for now.

To be honest I have no idea what you mean links/routing...